books/free-programming-books-langs.md

```diff
@@ -1115,6 +1115,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [JavaScript Challenges Book](https://tcorral.github.io/javascript-challenges-book/) - Tomás Corral Casas (HTML)
 * [JavaScript ES6 and beyond](https://github.com/AlbertoMontalesi/JavaScript-es6-and-beyond-ebook) - Alberto Montalesi (PDF, epub)
 * [JavaScript For Beginners](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
+* [JavaScript For Beginners](https://www.pdfdrive.com/javascript-javascript-for-beginners-learn-javascript-programming-with-ease-in-half-the-time-everything-about-the-language-coding-programming-and-web-pages-you-need-to-know-e195090778.html) - Stephen Blumenthal
```

Comment:
Remove this. We do not support PDFDrive links. We only allow links from authorized sources